src/Behat/Behat/Gherkin/Specification/Locator/FilesystemFeatureLocator.php

```diff
@@ -131,13 +131,19 @@ private function findFeatureFiles($path)
             return array($absolutePath);
         }
 
-        $iterator = new RegexIterator(
-            new RecursiveIteratorIterator(
-                new RecursiveDirectoryIterator($absolutePath)
-            ), '/^.+\.feature$/i',
+        $RDI = new RecursiveDirectoryIterator(
+            $absolutePath,
+            RecursiveDirectoryIterator::FOLLOW_SYMLINKS |
+            RecursiveDirectoryIterator::SKIP_DOTS
+        );
+        $RII = new recursiveIteratorIterator($RDI);
```

Comment:
You could simply construct them without any other variable like it was before btw.